File: KinderGartenManagment.Api/KinderGartenManagment.Api/Models/Payement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KinderGartenManagment.Api.Models
{
    public class Payement
    {
        public int Id { get; set; }
        public int ReceiptNumber { get; set; }
        public virtual ICollection<Modalite> Modalites { get; set; }
        public virtual ICollection<PayementEnrollement> PayementEnrollements { get; set; }
    }
}

File: KinderGartenManagementAngular/src/app/_core/_models/payement.ts
import { Modalite } from './modalite';
import { PayementEnrollement } from './payement-enrollement';

export class Payement {
    receiptnumber:number;
    payementenrollementmodels:PayementEnrollement[];
    modalitemodels:Modalite[];
    constructor(){
        this.modalitemodels=[new Modalite()];
    }
}
